src/Entities/Schema.php

```diff
@@ -177,9 +177,24 @@ public function __construct()
             'sessions/{sessionId}' => function (array $content) {
                 return new Session($content);
             },
-            'login' => function (array $content) {
+            'users/login' => function (array $content) {
```

Comment:
so, can we use `POST sessions` to create new session? To avoid verbs in URLs, as is recommended in REST?

Btw, seems unnecessary factory for `users/login`